README.md

```diff
@@ -246,6 +248,8 @@ Translations of the guide are available in the following languages:
   As far as embedded expressions go, there are also two acceptable
   options:
 
+  **Skookum uses the first option, no spaces in string embedded expressions.**
```

Comment:
If we are to not use spaces in string embedded expressions, should there be a rule to restrict how complex those expressions can get? I personally like this:

```ruby
"Tomorrow is #{Date.today + 1}"
```

over this:

```ruby
"Tomorrow is #{Date.today+1}"
```

I find spaces make it more readable, but I don't feel that strongly to argue against the rest of the team if everyone prefers not using spaces.